target/check-for-changes.sh

```diff
@@ -79,8 +79,15 @@ do
         esac
       done
 
-      # regenerate postix aliases
+      # Generate default /etc/aliases
       echo "root: ${PM_ADDRESS}" >/etc/aliases
+      echo "amavis: ${PM_ADDRESS}" >>/etc/aliases
+      if [[ ${ENABLE_CLAMAV} -1 1 ]]
```

Comment:
this is invalid, you accidentally used `-1` instead of `-eq` :)

Reply:
OMG